### PR TITLE
NFTs 2.0 on Westmint

### PR DESCRIPTION
**Pull request by jsidorenko**
This PR adds the new nfts pallet to westmint config

### CODE REVIEW

File: parachains/runtimes/assets/westmint/src/lib.rs [hunk 1]
```diff
@@ -541,6 +560,40 @@ impl pallet_uniques::Config for Runtime {
...
+	type ItemDeposit = ItemDeposit;
+	type MetadataDepositBase = UniquesMetadataDepositBase;
+	type AttributeDepositBase = AttributeDepositBase;
+	type DepositPerByte = DepositPerByte;
+	type StringLimit = UniquesStringLimit;
+	type KeyLimit = KeyLimit;
+	type ValueLimit = ValueLimit;
+	type ApprovalsLimit = NftsApprovalsLimit;
+	type ItemAttributesApprovalsLimit = NftsItemAttributesApprovalsLimit;
+	type MaxTips = NftsMaxTips;
```
Comment (gilescope) [inline parachains/runtimes/assets/westmint/src/lib.rs:589]: This can be 'ConstU32<10>' rather than needing a param_type ?

**Review by gilescope — APPROVED** [review]
Nice

File: parachains/runtimes/assets/westmint/src/lib.rs (hunk 1 above)
Comment (jsidorenko) [reply parachains/runtimes/assets/westmint/src/lib.rs:589]: sure

**Comment by jsidorenko** [comment]
/cmd queue -c bench-bot $ pallet westend-dev runtime_parachains::hrmp

**Comment by jsidorenko** [comment]
/cmd queue -c bench$ pallet westend-dev runtime_parachains::hrmp

**Comment by jsidorenko** [comment]
/cmd queue -c bench $ pallet westend-dev runtime_parachains::hrmp

**Review by joepetrowski — APPROVED** [review]

**Comment by joepetrowski** [comment]
/cmd queue -c bench $ pallet westend-dev pallet_nfts

**Review by ruseinov — APPROVED** [review]

**Comment by jsidorenko** [comment]
bot merge

**Pull request merged**